tests/ckeditor/basepath.js

```diff
@@ -31,6 +32,20 @@ describe( 'basePath', function() {
 		window.CKEDITOR_BASEPATH = 'http://foo.com/ckeditor/';
 		expect( CKEDITOR._getBasePath() ).to.equal( 'http://foo.com/ckeditor/' );
 	} );
+
+	function testGetBasePathFromTag( url, expectedBasePath ) {
```

Comment:
I don't understand. All the `it()` calls should add test cases and Bender.js should be able to filter them. Regardless of whether they are dynamically created or not. Actually... every single TC is created dynamically, because every single TC is created inside `describe()`'s callback.